collect.py

```diff
@@ -97,6 +97,7 @@ def get_all_attributes(driver, child):
 
 def do_something(driver, elem_attributes=None):
     elem = None
+    child_attributes = None
```

Comment:
Don't add this, but make the function return `elem_attributes` and rename `child_attributes` to `elem_attributes` at line 114.